README.md

```diff
@@ -1 +1,6 @@
-# Wear
\ No newline at end of file
+# Wearables
+
+## Wearable device sources
+Documentaton for running wearable data source can be found at the following links:
+- [FTShoes](https://github.com/robotology/wearables/blob/master/doc/How-to-run-FTshoes.md)
```

Comment:
If you do not hardcode the absolute URL, you ensure that the link will work correctly on branches/forks
```suggestion
- [FTShoes](doc/How-to-run-FTshoes.md)
```